static/app/views/eventsV2/queryList.tsx

```diff
@@ -103,34 +103,56 @@ class QueryList extends React.Component<Props> {
     const defaultTableFields = eventView.fields.map(({field}) => field);
     const {columns, aggregates} = getColumnsAndAggregates(defaultTableFields);
     const sort = eventView.sorts[0];
+
+    let yAxis = ['count()'];
+    if (typeof savedQuery?.yAxis === 'string') {
+      yAxis = [savedQuery.yAxis];
+    } else if (savedQuery?.yAxis) {
+      yAxis = savedQuery.yAxis;
+    } else if (eventView.yAxis) {
+      yAxis = [eventView.yAxis];
+    }
```

Comment:
this is a little difficult to ready. should we create a function instead?

Reply:
I've removed this and I'm doing something different with the rebase. Now we either pass `savedQuery?.yAxis` or `eventView.yAxis`